content/telegraf/v1.23/release-notes-changelog.md

```diff
@@ -10,6 +10,57 @@ menu:
     name: Release notes
     weight: 60
 ---
+## v1.23.3 [2022-08-17]
```

Comment:
Should be `v1.23.4` and could we change the date to [2022-08-16] to match the GitHub changelog?